### PR TITLE
Make categories more representative of language, add "tie up," and define colors

### PR DESCRIPTION
**Pull request by Property404**

### CODE REVIEW

**Comment by kutsan** [comment]
Thanks! I also gave you push access invitation, you can push whatever you want freely. I don't mind anything from the creator of the language.

File: syntax/fetlang.vim [hunk 1]
```diff
@@ -86,6 +81,11 @@ highlight link fetlangSpecial Special
 
 " }}}
 
+highlight Comment ctermfg=LightGreen cterm=none
+highlight Keyword ctermfg=DarkBlue cterm=bold
+highlight Function ctermfg=cyan term=none
+highlight Special cterm=bold
```
Comment (kutsan) [inline syntax/fetlang.vim:87]: I merged but I think we should remove color definitions. This is not a color scheme and it should not define any its colors. If you want `Keyword`s are blue, that is almost belongs to your `~/.vim/colors/` directory.

File: syntax/fetlang.vim [hunk 2]
```diff
@@ -2,50 +2,45 @@ if exists('b:current_syntax')
...
+syntax match fetlangFunction '\v\c<serve>'
 
 highlight link fetlangFunction Function
 
-" -- Repeat {{{1
+" -- Keyword {{{1
 " --------------------------------------------------------------------------------------------------
 
-syntax match fetlangRepeat '\v\c<while>'
-syntax match fetlangRepeat '\v\c<until>'
```
Comment (kutsan) [inline syntax/fetlang.vim:40]: `Repeat` category is special only for that kind of commands. Do you still think they should be under `Keyword`?

File: syntax/fetlang.vim (hunk 1 above)
Comment (Property404) [reply syntax/fetlang.vim:87]: I've never used vimscript so I don't really know anything, but before I added those, several of the categories were showing up as the same color. Is there another fix?

File: syntax/fetlang.vim (hunk 2 above)
Comment (Property404) [reply syntax/fetlang.vim:40]: Again, never used vimscript, but when I write C in vim, while and if seem to be the same color. You can go ahead and revert it.

File: syntax/fetlang.vim (hunk 1 above)
Comment (kutsan) [reply syntax/fetlang.vim:87]: You can `:help group-name` to look up syntax group names. If they are showing up the same color, that's about your color scheme. You should change your color scheme or modify it. Some color schemes prefer to show some syntax groups with same color, that's their problem.

File: syntax/fetlang.vim (hunk 2 above)
Comment (kutsan) [reply syntax/fetlang.vim:40]: That's okay. If that's the case, i'll make adjustments.

File: syntax/fetlang.vim (hunk 1 above)
Comment (Property404) [reply syntax/fetlang.vim:87]: Gotcha, thanks

Comment (kutsan) [reply syntax/fetlang.vim:87]: By the way, vim has bunch of built-in color schemes, you can use them for testing purposes, write `:colorscheme` and hit <kbd>Tab</kbd>, vim will auto complete available ones. _You may need to `:set wildmenu` for auto completion._

**Pull request merged**